config/config.go

```diff
@@ -101,7 +101,6 @@ func newDefaultBootstrapConfig() *BootstrapConfig {
 // MiningConfig holds all configuration options related to mining.
 type MiningConfig struct {
 	MinerAddress            address.Address `json:"minerAddress"`
```

Comment:
It was decided that blockSignerAddress is not only redundant (use the miner owner public key instead which is already stored), but does not belong in config.